app/src/test/java/nodomain/freeyourgadget/gadgetbridge/test/LanguageUtilsTest.java

```diff
@@ -61,7 +61,7 @@ public void testStringTransliterateBengali() throws Exception {
         //input with cyrillic and diacritic letters
         String input = "অনিরুদ্ধ";
         String output = LanguageUtils.transliterate(input);
-        String result = "oniruddho";
+        String result = "aniruddha";
```

Comment:
Can you please add some more testcases? Since the regular expression handling is quite involved, I would like to have a few matching and partly matching testcases.